plugin.video.rocketbeans/resources/lib/plugin.py

```diff
@@ -20,149 +20,99 @@
 
 
 def getString(string_id):
-    return __addon__.getLocalizedString(string_id).encode('utf-8', 'ignore')
+    return __addon__.getLocalizedString(string_id)
 
 
 def get_livestream():
     if __addon__.getSetting("stream") == 0:
         t = TwitchStream(config.TWITCH_USER_LOGIN)
-        livestream_url, title, livestream_thumbnail = t.url, t.title, t.thumbnail
+        return t.url, t.title, t.thumbnail
     else:
-        livestream_url, title, livestream_thumbnail = YoutubeStream(
-        ).get_live_video_info_from_channel_id(config.CHANNEL_ID)
-    return livestream_url, livestream_thumbnail, title
+        return YoutubeStream().get_live_video_info_from_channel_id(config.CHANNEL_ID)
 
 
-@plugin.route('/')
+@plugin.route("/")
 def index():
     livestream_url, livestream_thumbnail, title = get_livestream()
-    li = createListItem(
-        ('Live | %s' % title),
-        livestream_thumbnail,
-        True,
-        getString(32001),
-        0
-    )
-    addDirectoryItem(
-        plugin.handle,
-        livestream_url,
-        li
-    )
+    if livestream_url:
+        li = createListItem(
+            ("Live | %s" % title), livestream_thumbnail, True, getString(32001), 0
```

Comment:
Missing localization for Live?

Reply:
Yeah, but as all content is german anyway it doesn't really matter :/

Reply:
Sure but if you provide localization support there's no reason for some of the ones presented on the UI to be forced to english. Since this was not something introduced in this PR it's fine to me but please keep it on the todo list for a next update